src/dom/dom-manager.js

```diff
@@ -1,3 +1,12 @@
+/**
+ * ## DOM Manager
+ *
+ * The Flow.js DOM Manager controls how different DOM elements -- including the attributes and attribute handlers provided by Flow.js for [variables](../../attributes-overview/), [operations](../../operations-overview/) and [conversion](../../converter-overview/), and those [you create](./attributes/attribute-manager/) -- are bound to the variable and operations [channels](../../channel-overview/) to link them with your project's model. See the [Epicenter architecture details](../../../creating_your_interface/arch_details/) for a visual description of how the DOM Manager relates to the [rest of the Epicenter stack](../../../creating_your_interface/).
+ *
+ * The DOM Manager as written is specific to Epicenter and jquery. Although largely 'invisible,' it is the piece that you need to replace if you want to extend Flow.js to work with other technologies, like [Backbone.js](http://backbonejs.org) or [Angular.js](https://angularjs.org). [Contact us](http://forio.com/about/contact/) if you are interested in extending Flow.js in this way -- we'll be happy to talk about it in more detail.
```

Comment:
The DOM manager is an integral part of the Flow.js architecture but, in keeping with our general philosophy of extensibility and configurability, it is also replaceable. If you want to manage your DOM state with Backbone Views for instance, or Angular, while still using the Channels to handle the communication with your model, this is the piece you'd replace. [Contact us](http://forio.com/about/contact/) if you are interested in extending Flow.js in this way -- we'll be happy to talk about it in more detail.

<Very awkwardly worded above ^. Mostly typing as I think. Feel free to edit/replace  >